pkg/syncer/metadatasyncer.go

```diff
@@ -42,6 +42,7 @@ import (
 	restclient "k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/util/workqueue"
+	"k8s.io/cri-api/pkg/errors"
```

Comment:
we generally uses "k8s.io/apimachinery/pkg/api/errors" to call IsNotFound

Reply:
Done